lib/js_routes.rb

```diff
@@ -19,7 +19,8 @@ class JsRoutes
     url_links: nil,
     camel_case: false,
     default_url_options: {},
-    compact: false
+    compact: false,
+    serializer: "this.default_serializer"
```

Comment:
I think we should use nil here. This value feels too magical if someone will try to debug it.